src/client/chrome/commands.js

```diff
@@ -0,0 +1,130 @@
+// @flow
+
+const {
+  toServerLocation,
+  fromServerLocation,
+  createLoadedObject,
+} = require("./create");
+
+import type { Location } from "../types";
+import type { ServerLocation, Agents } from "./types";
+
+type setBreakpointResponseType = {
+  breakpointId: string,
+  serverLocation?: ServerLocation,
+};
+
+let debuggerAgent;
+let runtimeAgent;
+let pageAgent;
+
+function setupCommands({ Debugger, Runtime, Page }: Agents) {
+  debuggerAgent = Debugger;
+  runtimeAgent = Runtime;
+  pageAgent = Page;
+}
+
+function resume() {
+  return debuggerAgent.resume();
+}
+
+function stepIn() {
+  return debuggerAgent.stepInto();
+}
+
+function stepOver() {
+  return debuggerAgent.stepOver();
+}
+
+function stepOut() {
+  return debuggerAgent.stepOut();
+}
+
+function pauseOnExceptions(
+  shouldPauseOnExceptions: boolean,
+  shouldIgnoreCaughtExceptions: boolean
+) {
+  if (!shouldPauseOnExceptions) {
+    return debuggerAgent.setPauseOnExceptions({ state: "none" });
+  }
+  const state = shouldIgnoreCaughtExceptions ? "uncaught" : "all";
+  return debuggerAgent.setPauseOnExceptions({ state });
+}
+
+function breakOnNext() {
+  return debuggerAgent.pause();
+}
+
+function sourceContents(sourceId: string) {
+  return debuggerAgent.getScriptSource({ scriptId: sourceId }).then(({
+    scriptSource,
+  }) => ({
+    source: scriptSource,
+    contentType: null,
+  }));
+}
+
+async function setBreakpoint(location: Location, condition: string) {
+  let {
+    breakpointId,
+    serverLocation,
+  }: setBreakpointResponseType = await debuggerAgent.setBreakpoint({
+    location: toServerLocation(location),
+    columnNumber: location.column,
+  });
+
+  const actualLocation = fromServerLocation(serverLocation) || location;
+
+  return {
+    id: breakpointId,
+    actualLocation: actualLocation,
+  };
+}
+
+function removeBreakpoint(breakpointId: string) {
+  return debuggerAgent.removeBreakpoint({ breakpointId });
+}
+
+async function getProperties(object: any) {
+  const { result } = await runtimeAgent.getProperties({
+    objectId: object.objectId,
+  });
+
+  const loadedObjects = result.map(createLoadedObject);
+
+  return { loadedObjects };
+}
+
+function evaluate(script: string) {
```

Comment:
could we rename `script` to `expression` so it matches what we passe in `evaluate` ?